content/swiftui/concepts/views/terms/button/button.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'Button'
+Description: 'Used to perform certain actions.'
+Subjects:
+  - 'Sotfware Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **button** is a basic component in the [UI](https://www.codecademy.com/resources/docs/uiux), which the user interacts with to perform a certain action, such as logging into a site, or deleting a file etc.
+
+## Syntax
+
+```pseudo
+Button("Label of button") {
+  // What happens when you click the button
+}
+```
+
+The `"Label of button"` is what is shown on the button.
```

Comment:
Same question here! Can we include a line about the possibilities of what can appear on the button. So far, we know that text can appear. Look to the apple docs if you need some insight: https://developer.apple.com/documentation/swiftui/button

Reply:
Added about this one.
please check